### PR TITLE
Fix Point Type when column value is null

### PR DESCRIPTION
**Pull request by janjitsu**

### CODE REVIEW

File: src/PointType.php [hunk 1]
```diff
@@ -36,8 +36,12 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      * @param AbstractPlatform $platform
      * @return Point
      */
-    public function convertToPHPValue($value, AbstractPlatform $platform): Point
```
Comment (vinyvicente) [inline src/PointType.php:39]: @janjitsu ,

Keep the return value. The lib requires php >= 7.0

php 5 die.

Comment (vinyvicente) [reply src/PointType.php:39]: Can be upgraded to php7.1 with ?Point

**Comment by evalexpr** [comment]
Could we get this merged? I'm having the same issue

**Comment by vinyvicente** [comment]
@janjitsu awaiting

**Comment by vinyvicente** [comment]
@janjitsu @W1lkins  updated library. Enjoy.

**Pull request closed without merging**